webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-grading-comment.js

```diff
@@ -128,12 +125,14 @@ export class SakaiRubricGradingComment extends RubricsElement {
         height: 40
       });
 
-      commentEditor.on('change', () => this.criterion.comments = commentEditor.getData());
```

Comment:
This change event is no longer necessary because `this.criterion.comments` will be updated in `blur` event. If we maintain this line, `this.criterion.comments` will be always up to date and it will be not possible to check if the var was updated or not.